parsing/printast.ml

```diff
@@ -51,15 +51,14 @@ let fmt_string_loc f x =
 
 let fmt_constant f x =
   match x with
-  | Const_int (i) -> fprintf f "Const_int %d" i;
-  | Const_char (c) -> fprintf f "Const_char %02x" (Char.code c);
-  | Const_string (s, None) -> fprintf f "Const_string(%S,None)" s;
-  | Const_string (s, Some delim) ->
+  | PConst_int (i,None) -> fprintf f "Const_int %s" i;
```

Comment:
Given that the format here is apparently meant to give re-parseability, you should reprint `PConst` rather than `Const`.